PRIME1/solution.ex

```diff
@@ -0,0 +1,47 @@
+defmodule ConversionUtils do
+  def convert_to(s, :string) do
+    Integer.parse(s)
+    |> elem(0)
+  end
+end
+
+defmodule IOUtils do
+  def parse_input_line(l) do
+    String.trim(l)
+    |> String.split(" ")
+    |> Enum.map(&ConversionUtils.convert_to(&1, :string))
+  end
+end
+
+defmodule ListUtils do
+  def print_elements(l) do
+    Enum.each l, &IO.puts(&1)
+    IO.puts ''
+  end
+end
+
+defmodule PrimeFinder do
+  defp _is_divided_by?(i, n) when i * i > n, do: false
+  defp _is_divided_by?(i, n) when rem(n, i) == 0, do: true
+  defp _is_divided_by?(2, n), do: _is_divided_by? 3, n
+  defp _is_divided_by?(3, n), do: _is_divided_by? 5, n
+  defp _is_divided_by?(i, n) when rem(i, 6) == 1, do: _is_divided_by? i + 4, n
+  defp _is_divided_by?(i, n) when rem(i, 6) == 5, do: _is_divided_by? i + 2, n
+
+  defp _has_divisors?(n) do
+    _is_divided_by? 2, n
+  end
+
+  def find(l) do
+    [m, n] = IOUtils.parse_input_line l
+    for i <- m..n do
```

Comment:
What do you think about change this `for` to a `Enum.each` function

```elixir
Enum.each(m..n, &_print_prime(&1))
```

Reply:
You can try that and check if the execution time is reduced. I'm not sure it will.